File: Leetcode/717. 1-bit and 2-bit Characters/solution.py
class Solution:
    def isOneBitCharacter(self, bits: List[int]) -> bool:
        res = False
        i = 0
        while i < len(bits):
            if bits[i] == 0:
                res = True
                i += 1
                continue
            
            res = False
            i += 2
        
        return res

File: Leetcode/717. 1-bit and 2-bit Characters/solution.cpp
class Solution {
public:
    bool isOneBitCharacter(vector<int>& bits) {
        int i = 0;
        bool result = false;

        while( i < bits.size()) {
            if (bits[i] == 0)
            {
                result = true;
                i += 1;
                continue;
            }

            result = false;
            i += 2;
        }

        return result;
    }
};